src/controllers/expenses.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const userService = require('../services/users.js');
+const expensesService = require('../services/expenses.js');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  const expenses = expensesService.getAll({
```

Comment:
you don't have to name every property when the whole payload is meant to be send, use just req.body here